src/ucs/sys/sock.c

```diff
@@ -230,6 +230,82 @@ int ucs_socket_is_connected(int fd)
     return 1;
 }
 
+
+UCS_F_NOOPTIMIZE ucs_status_t   /* GCC failed to compile it in release mode */
```

Comment:
need better way to fix it

Reply:
the compiler doesn't say what the error is. what would be a better way to fix?
when is it allowed to use this macro then?

Reply:
i'm pretty sure the error does not show because of various flags passed by makefile
need to isolate the gcc command and find the real error
this flag should be used when the code generated by optimization is not correct